driver/razermouse_driver.c

```diff
@@ -3680,6 +3680,8 @@ static DEVICE_ATTR(backlight_led_state,            0660, razer_attr_read_backlig
 #define BIT_TILT_L 5
 #define BIT_TILT_R 6
 
+#define SCROLL_DETENT 120
```

Comment:
What's this magic value?

Reply:
> The accumulated value 120 represents movement by one detent

https://www.kernel.org/doc/html/latest/input/event-codes.html#ev-rel

Reply:
Why are we pretending to have high-resolution scrolling support now though? Afaik we don't get smaller increments from the hardware so there's not really a point in pretending?

Reply:
mostly just added it because the generic HID driver reports these events for the same hardware.  The same goes for the Logitech mouse I have and its driver, despite it also not reporting anything other than +/-120.   
Figured is some piece of software was expecting these events instead of both its work around their bug.

I don't mind removing that commit if you'd prefer though.

Reply:
Looked into this a little bit more and it [looks like just the `REL_WHEEL_HI_RES` event is configured in the default HID driver](https://github.com/torvalds/linux/blob/master/drivers/hid/hid-input.c#L750), Then [it calculates the `REL_WHEEL` from the `REL_WHEEL_HI_RES` and emits both events](https://github.com/torvalds/linux/blob/master/drivers/hid/hid-input.c#L1309). This seems to work as it appears that Windows sets the expectation of 120 per detent from the hardware.

Reply:
I removed this contentious part of this PR for a later one

Reply:
Thanks